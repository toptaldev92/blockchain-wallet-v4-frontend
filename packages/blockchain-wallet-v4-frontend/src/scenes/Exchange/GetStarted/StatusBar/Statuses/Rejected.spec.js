import { Rejected } from './Rejected'
import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

jest.mock('blockchain-info-components', () => ({
  Button: 'button',
  ICon: 'icon'
}))

describe('Rejected', () => {
  it('renders correctly', () => {
    const component = shallow(<Rejected />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
