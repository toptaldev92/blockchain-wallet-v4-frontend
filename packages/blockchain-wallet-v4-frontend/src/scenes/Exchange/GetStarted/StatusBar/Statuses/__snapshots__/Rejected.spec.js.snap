// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rejected renders correctly 1`] = `
<styled.div>
  <Component
    color="white"
    size="14px"
    weight={500}
  >
    <FormattedMessage
      defaultMessage="Account Verification Failed"
      id="scenes.exchange.getstarted.status.rejected.title1"
      values={Object {}}
    />
  </Component>
  <Component
    size="14px"
    weight={400}
  >
    <FormattedMessage
      defaultMessage="Unfortunately we had some trouble with the documents that you’ve supplied and we can’t verify your account at this time. "
      id="scenes.exchange.getstarted.status.rejected.description"
      values={Object {}}
    />
  </Component>
</styled.div>
`;
